dask_conversions/done/Dask_water_permamency_L8.ipynb

```diff
@@ -4,7 +4,39 @@
    "cell_type": "code",
```

Comment:
<p>un-comment this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>